app/assets/styles/styles.css

```diff
@@ -1,6 +1,9 @@
 
 /* The body has a default margin, which we do not want interfering with our image header, which is why we have set it to 0. The reason that we have set the font-family to sans-serif is so that you’re not stuck with Times New Roman */
 
+* {
+  box-sizing: border-box;
+}
```

Comment:
We should not need this. Did you make sure to base your branch on the most current work in `release-branch-v1`?

Reply:
Yes. It was not up-dated.

My intention was to understand why MR was having problem with this rule. 

It is now removed.